pkg/api/v1/zz_generated.defaults.go

```diff
@@ -136,9 +136,6 @@ func SetObjectDefaults_PersistentVolume(in *v1.PersistentVolume) {
 	if in.Spec.PersistentVolumeSource.HostPath != nil {
 		SetDefaults_HostPathVolumeSource(in.Spec.PersistentVolumeSource.HostPath)
 	}
-	if in.Spec.PersistentVolumeSource.RBD != nil {
-		SetDefaults_RBDVolumeSource(in.Spec.PersistentVolumeSource.RBD)
```

Comment:
need to copy `SetDefaults_RBDVolumeSource` to `SetDefaults_RBDPersistentVolumeSource`

Reply:
which also means copying the registered fuzzing function in pkg/api/fuzzer/fuzzer.go for RBDVolumeSource for RBDPersistentVolumeSource and registering the new type in pkg/api/testing/defaulting_test.go#TestDefaulting

Reply:
Done fuzzer, but could you clarify about the first part? Do you want me to manually edit generated file or there is another way to accomplish it without editiing zz file.

Reply:
SetDefaults_RBDVolumeSource is not in a generated file. Duplicate it there for the new type and rerun generation scripts which will fix up this file

Reply:
Done, I think at least, please check when you have a second.